database/README.md

```diff
@@ -0,0 +1,14 @@
+This document explains the database that the orderbook api uses.
+
+We need to store orders persistently even if the orderbook service goes. For this we use a PostgreSQL database. The schema can be found in `schema.sql`.
+
+In addition to the orders, we also store trade events. This allows us to keep track of filled amounts.
+
+To prevent users from spamming the database we should when receiving an order check that
+* the erc20 spending is approved
```

Comment:
For the database it won't matter if the user places the same order again because the primary key is the uid which will be the same. So there can only be that one entry. We will probably detect this when in our create-order endpoint and return an appropriate error message.